wordpress-mobile/WordPress-iOS
# Update Reader tab bar icon and tab bar colors

Fixes #8005, refs #8004.

This PR makes a couple of changes to the tab bar and its icons to improve the appearance:

* Updated the Reader icon as per #8005.
* All of the icons (with the exception of Unread Notifications, which requires two colors) are now set as `template` images instead of `original`, and are tinted at runtime.
* The Write icon has been updated to include a transparent icon in the center (previously it was filled white, so didn't work as a template image).
* The icon and text colors have been updated to match their intended colours:
   * Unselected icons are `greyLighten10`.
   * Unselected text is `grey`.
   * Selected icons are `mediumBlue`
   * Selected text is `wordPressBlue`.

![simulator screen shot - iphone 8 - 2017-10-20 at 21 25 29](https://user-images.githubusercontent.com/4780/31840945-5402ad0a-b5df-11e7-8974-14e9c81c332d.png)

To test:

* Build and run, check that the icons and colors match those described.
* Also check that the Unread Notifications icon matches.

Needs review: @elibud 
cc @mattmiklic
